IBSWeb/wwwroot/css/modern-navbar.css

```
@@ -0,0 +1,943 @@
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+}
+
+.mm-item .mnav-col-item-title,
+.mm-trigger .mnav-col-item-title {
+    font-size: 13.5px;
+    font-weight: 500;
+    color: var(--mnav-on-primary);
+    white-space: normal;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)